docs/help/account/github-student.md

```diff
@@ -0,0 +1,17 @@
+# GitHub Student Developer Pack
+
+The [GitHub Student Developer Pack](https://education.github.com/pack?sort=popularity&tag=Infrastructure+%26+APIs) is a collection of tools available for free to eligible students, enabling them to try out professional tools that would otherwise be financially out of reach.
+
+[Bump.sh](http://bump.sh/) is incredibly proud to be a part of this offering. We have been students too and understand the importance of good documentation for your academic or personal projects.
+
+To benefit from this, you must be an eligible student (see conditions on GitHub) and use your GitHub account to sign up for [Bump.sh](http://bump.sh/). You can enjoy your [Bump.sh](http://bump.sh/) subscription for free as long as you are recognized as a student on your GitHub account.
```

Comment:
```suggestion
To benefit from this, you must be an eligible student (see conditions on GitHub) and use your GitHub account to sign up to [Bump.sh](http://bump.sh/). You can enjoy your [Bump.sh](http://bump.sh/) subscription for free as long as you are recognized as a student on your GitHub account.
```

Reply:
That's way more complex than that. But let's go with 'to'